### PR TITLE
readme update with ui deploy instructions

### PR DESCRIPTION
**Pull request by bvennam**

### CODE REVIEW

**Comment by bvennam** [comment]
@jasonpet can you review when you get a chance?

File: README.md [hunk 1]
```diff
@@ -88,11 +88,31 @@ deploy.sh
 ```
 
 # Deploy Cloud Object Storage Package with IBM Cloud Console
-Currently there is no direct way to deploy the Cloud Object Storage package from the IBM Cloud Functions UI.  However, there is a workaround that provides you with a way to install the cloud-object-storage package in the NodeJS runtime. 
 
-To do this you will have to install the **Upload Image** template which installs the **cloud-object-storage** package as a prerequisite for the template. 
+## Deploy Package from the UI
+In the Cloud Functions console, go to the Create page External link icon.
```
Comment (Rachael-Graham) [inline README.md]: `go to the [Create page](https://console.bluemix.net/openwhisk/create).`

File: README.md [hunk 2]
```diff
@@ -88,11 +88,31 @@ deploy.sh
...
+
+Once the Package has been installed you will be redirected to the Actions page and can search for your new Package, which is named cloud-object-storage.
+
+##Bind Service Credentials
+
+To use the Actions in the cloud-object-storage Package, you must bind service credentials to the actions.
+
+To bind service credentials to all actions in the package, follow steps 5 and 6 in the CLI instructions listed above.
+
+To bind service credentials to individual actions, complete the following steps in the UI. Note: You must complete the following steps for each action that you want to use.
```
Comment (Rachael-Graham) [inline README.md]: I would combine the three sentences in lines 107 through 111 into one sentence:

To use the actions in the cloud-object-storage package, you must bind service credentials to the actions.  Note: You must complete the following steps for each action that you want to use.

File: README.md [hunk 3]
```diff
@@ -88,11 +88,31 @@ deploy.sh
...
+
+Click Install Packages.
+
+Click on the IBM Cloud Object Storage Package group, and then click on the IBM Cloud Object Storage Package.
+
+In the Available Runtimes section, select either NodeJS or Python from the drop-down list and then click Install.
+
+Once the Package has been installed you will be redirected to the Actions page and can search for your new Package, which is named cloud-object-storage.
+
+##Bind Service Credentials
```
Comment (Rachael-Graham) [inline README.md]: add a space (I think it won't render correctly w/o a space)

**Review by Rachael-Graham — COMMENTED** [review]
Small changes, otherwise good!

**Comment by bvennam** [comment]
@Rachael-Graham thanks for the feedback -- i just made those updates.

**Review by Rachael-Graham — APPROVED** [review]
LGTM!

**Review by jasonpet — APPROVED** [review]
LGTM

**Pull request merged**